samples/Catalog/Catalog.ios.js

```diff
@@ -602,6 +603,46 @@ class ProgrammaticAnnotations extends Component {
               title="addAnnotation"
             />
           </View>
+          <View>
+            <Button
+              onPress={() => {
+                // Programmatically remove an existing ink annotation.
+                const annotationJSON = {
```

Comment:
I'd generalize this example and query all annotations from the page and remove the first one, instead of hardcoding an annotation to be removed.